redact_sample.py

```diff
@@ -0,0 +1,120 @@
+import argparse
+import csv
+import logging
+import os
+import sys
+from typing import Iterable
+
+from generate_sample_file import SampleGenerator
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
+SampleGen = SampleGenerator()
+sensitive_estab_types = ['MILITARY US SFA', 'MILITARY SFA', 'MILITARY US SLA', 'ROYAL HOUSEHOLD',
```

Comment:
should be named uppercase to denote a 'constant'? I'm not clear on the Python naming conventions

Reply:
Clearly the convention exists within the repo, so I will make this change - just waiting to see if any other changes are necessary so as to not flood commits.